index.bs

```diff
@@ -2778,6 +2831,276 @@ The [=remote end steps=] with <var ignore>session</var> and |command parameters|
 
 1. Return [=success=] with data null.
 
+</div>
+#### The browsingContext.locateNodes Command ####  {#command-browsingContext-locateNodes}
+
+The <dfn export for=commands>browsingContext.locateNodes</dfn> command returns a
+list of all nodes matching the specified locator.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      browsingContext.LocateNodes = (
+        method: "browsingContext.locateNodes",
+        params: browsingContext.LocateNodesParameters
+      )
+
+      browsingContext.LocateNodesParameters = {
+         context: browsingContext.BrowsingContext,
+         locator: browsingContext.Locator,
+         ? maxReturnedNodeCount: (js-uint .ge 1),
+         ? ownership: script.ResultOwnership,
+         ? serializationOptions: script.SerializationOptions
```

Comment:
I think we also need to allow passing in a sandbox name, so that you can create the handles in the right sandbox.

Reply:
So added.